### PR TITLE
Adding "asdf" line

### PR DESCRIPTION
**Pull request by annachelondaeva**

### CODE REVIEW

**Comment by annachelondaeva** [comment]
Close #7

File: doc.md [hunk 1]
```diff
@@ -10,4 +10,6 @@ Change 3
 
 Change 4
 
-Change 5
\ No newline at end of file
+Change 5
+
+asdf
```
Comment (thorny-thorny) [inline doc.md]: Move the "asdf" right below "qweqwe"

**Review by thorny-thorny — CHANGES_REQUESTED** [review]
See my comment about "asdf" line

File: doc.md (hunk 1 above)
Comment (annachelondaeva) [reply doc.md]: Done

**Review by thorny-thorny — APPROVED** [review]

**Pull request merged**